release.config.js

```diff
@@ -166,25 +171,23 @@ if (manifestExists && GITHUB_REF === "refs/heads/main") {
   });
 }
 
-if (process.env.SKIP_NPM_PUBLISH === undefined) {
```

Comment:
I don't think we want this `if` conditional removed: this was intentional in https://github.com/open-sauced/release/pull/22 since most of these asset files being committed are artifacts from an NPM release (like a `package.json`, `yarn.lock`, the `public/` directory, etc.)

If this check is removed, it'll break [usage for `open-sauced/pizza`](https://github.com/open-sauced/pizza/blob/beta/.github/workflows/release.yaml)

Reply:
The package files are only updated by the `npm` plugin, which now has a conditional and this conditional prevents the `CHANGELOG` from being committed as well if `SKIP_NPM_PUBLISH` is opted into.